docs/Walkthrough.md

```diff
@@ -225,7 +225,19 @@ You will now have a new tarball in `rel/<name>/releases/<upgrade_version>/` whic
 
 ## Deploying an upgrade
 
-This part is very straightforward:
+*Important*: This part is very straightforward with an important
```

Comment:
@bitwalker This was in the `Use With Phoenix` document. I ran into this(the marked "old" issue) while experimenting in my test application. If this requires further discussion I can open a new issue.